### DISCUSSION TITLE
ERROR (set_mesh): The new mesh has a different number of vertices/faces. Please clear the mesh before plotting.

### DISCUSSION THREAD

**Discussion by sukamay** [libigl/libigl#1809] [Q&A]
when I try to change the number of vertices and call `set_mesh` function, I will get this error. I try to add `data.clear()` before `data.set_mesh(V, F)`, but I still get this error. How can I do if I want to change the size of vertices during running time?

**Comment by jdumas** [comment]
Calling `.clear()` before `.set_mesh()` should resolve the issue. Can you share a minimal working example + mesh to reproduce the issue?
